packages/graph/hash_graph/lib/graph/src/store/postgres/pool.rs

```diff
@@ -17,16 +33,18 @@ impl PostgresStorePool {
     /// - if creating a connection returns an error.
     pub async fn new(db_info: &DatabaseConnectionInfo) -> Result<Self, StoreError> {
         tracing::debug!("Creating connection pool to Postgres");
-        let mut connection_options = PgConnectOptions::default()
-            .username(db_info.user())
+        let mut config = Config::new();
+        config
+            .user(db_info.user())
             .password(db_info.password())
             .host(db_info.host())
             .port(db_info.port())
-            .database(db_info.database());
-        connection_options.log_statements(LevelFilter::Trace);
+            .dbname(db_info.database());
 
         Ok(Self {
-            pool: PgPool::connect_with(connection_options)
+            pool: Pool::builder()
+                .error_sink(Box::new(ErrorLogger))
+                .build(PostgresConnectionManager::new(config, NoTls))
```

Comment:
We probably do want TLS when in production. Could we add a TODO for figuring out how we do a produciton TLS setup later down the line?

Reply:
I can change the code to support TLS now if you like. I also don't mind adding a TODO.

Reply:
I added support for TLS: 18b641db8d60aebb96a25104e56bc6f0718b24eb